src/workbookManager.ts

```diff
@@ -120,20 +121,20 @@ export class WorkbookManager {
         while (tableColumns.lastChild) {
             tableColumns.removeChild(tableColumns.lastChild);
         }
-        var columnIndex = 1;
+        var columnIndex = 0;
         tableData.columnNames.forEach(columnName => {
```

Comment:
You can use the index provided by forEach, if not - leaving it as initial value '1' seems more clean.